### PR TITLE
Solid CR added. 45 degrees sym. Fully withdrawn

### PR DESCRIPTION
**Pull request by andrewryh**
- [x] Axial reflectors
- [x] 45 degree symmetry
- [x] Solid CR instead of hollow cylinder
- [x] B4C CR

### CODE REVIEW

File: solid-b4c-rods/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Several assumptions have been made due to lack of information.
+The density of the salt was determined using $`\rho(T)=6.105-0.001272T[K]`$
+which is based on https://aip.scitation.org/doi/10.1063/1.3253134 although the temperatures of interest are outside the support of the equation.
+
+The control rods are solid, made of B4C with reactivity worth 3.9%. Control rods are fully withdrawn (see lines 7-14 tap.ini).
+
+45 degrees symmetry with reflective boundary condition was used.
+
+10000 neutrons, 1000 active cycles, 100 inactive cycles gave STD+/-22pcm
+9 minutes
```
Comment (dogrady2) [inline solid-b4c-rods/README.md]: 1 core?

File: solid-b4c-rods/core.ini [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+%The CR is said to be in a voided thimble with similar design to MSRE
+%Assumed thickness/outer radius was constant for each material
+%Assumed GT took up  1 moderator rod pitch
+
+surf 120 cyl 0.0 0.0 1.5        %OR for  CR GT
+surf 121 cyl 0.0 0.0 1.425      %IR for  CR GT
+
+surf 122 cyl 0.0 0.0 0.855      %OR for CR clad
+surf 123 cyl 0.0 0.0 0.810      %OR for CR pois
+%surf 124 cyl 0.0 0.0 0.630     %OR for CR clad
```
Comment (dogrady2) [inline solid-b4c-rods/core.ini]: I have been "yelled" at before for leaving commented code, you should just delete the lines

**Review by dogrady2 — APPROVED** [review]
I think moving forward the structure of this repository needs to be changed such that it is easier to differentiate between the different geometries. I'm not sure what the best way to do this would be, branches or tags I assume. There is also a lot of commented lines in these files, not sure if they should be left in the commits or not.

**Comment by andrewryh** [comment]
> I think moving forward the structure of this repository needs to be changed such that it is easier to differentiate between the different geometries. I'm not sure what the best way to do this would be, branches or tags I assume. There is also a lot of commented lines in these files, not sure if they should be left in the commits or not.

I think multiple folders with README in each would work better than multiple branches. Later I gonna remove those `v1-1`, `v1-2` folder and the repository will be much cleaner.

File: solid-b4c-rods/README.md (hunk 1 above)
Comment (andrewryh) [reply solid-b4c-rods/README.md]: 12 threads, 6 cores. Added this in README

File: solid-b4c-rods/core.ini (hunk 2 above)
Comment (andrewryh) [reply solid-b4c-rods/core.ini]: Remove all commented lines, thanks.

**Comment by andrewryh** [comment]
@dogrady2 All fixed.
Please, check and merge.

**Review by dogrady2 — APPROVED** [review]

**Pull request merged**